FrontEnd/src/components/ProfileDetail/MainInfo/ProfileDetailNavBar.jsx

```diff
@@ -0,0 +1,48 @@
+import { HashLink } from 'react-router-hash-link';
+import classes from './ProfileDetailNavBar.module.css';
+
+function ProfileDetailNavBar() {
+  const menuItems = [
+    'Про компанію',
+    'Стартап',
+    'Товари / послуги',
+    'Логістика товарів / послуг',
+    'Формат співпраці',
+  ];
+  const menuLinks = [
+    'about-company',
+    'startup',
+    'products-services',
+    'logistics',
+    'cooperation',
+  ];
```

Comment:
Instead of using two arrays i would recommend to use array of objects
`
const menuItems = [
  {
    link: 'about-company',
    displayName: 'Про компанію'
  }, ...
]
`